platform/macos/export/export_plugin.cpp

```diff
@@ -2288,6 +2294,14 @@ Error EditorExportPlatformMacOS::export_project(const Ref<EditorExportPreset> &p
 			}
 		}
 
+		if (FileAccess::exists(ent_path)) {
+			print_verbose("entitlements:\n" + FileAccess::get_file_as_string(ent_path));
+		}
+
+		if (FileAccess::exists(hlp_ent_path)) {
+			print_verbose("helper entitlements:\n" + FileAccess::get_file_as_string(hlp_ent_path));
+		}
+
```

Comment:
I'm not sure how useful it is, but I guess it's OK for verbose mode.

Reply:
I was having issues debugging entitlement with my app.  Specifically, I was missing the entitlement that allowed me to "login with apple".  This was useful for me to see the entitlements I had and the entitlements I was missing.

But yes, I only added it in verbose mode so as not to clutter the console output in normal usage :)